hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/config/BrokerLatencyReporterConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package pl.allegro.tech.hermes.frontend.config;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import pl.allegro.tech.hermes.common.metric.MetricsFacade;
+import pl.allegro.tech.hermes.common.metric.executor.InstrumentedExecutorServiceFactory;
+import pl.allegro.tech.hermes.frontend.producer.BrokerLatencyReporter;
+
+import java.util.concurrent.ExecutorService;
+
+
+@Configuration
+@EnableConfigurationProperties(BrokerLatencyReporterProperties.class)
+public class BrokerLatencyReporterConfiguration {
+
+    @Bean
+    BrokerLatencyReporter brokerLatencyReporter(BrokerLatencyReporterProperties properties,
+                                                MetricsFacade metricsFacade,
+                                                InstrumentedExecutorServiceFactory executorServiceFactory) {
+        ExecutorService executorService = executorServiceFactory.getExecutorService(
+                "broker-latency-reporter",
```

Comment:
nit: what do You think about moving this string to the static field? or maybe it is a common pattern when creating executor service?

Reply:
I'd say in this case and in BrokerMetrics this is a matter of preference. These string are not used anywhere else + [there is no performance difference](https://stackoverflow.com/questions/36185564/java-strings-private-static-vs-local-variable-performance) so it does not really make a difference